include/ensmallen_bits/parallel_sgd/parallel_sgd.hpp

```diff
@@ -146,7 +147,7 @@ class ParallelSGD
 
   //! Controls whether or not the individual functions are shuffled when
   //! iterating.
-  bool shuffle;
+  bool shuffleFlag;
```

Comment:
I noticed that there are specific specializations in mlpack for `ParallelSGD` that use the `shuffle` member; so if we change the name, downstream mlpack will fail.  I don't mind if we want to do an `#ifdef` check on the ensmallen version downstream, but it would be better if we don't need to change this variable's name at all.  Do you think that's possible or is there a reason it needs to become `shuffleFlag`?

Reply:
Just answered my own question... it conflicts with `shuffle()`.  I know the downstream change to make; will open a PR there.